src/pybind/mgr/localpool/module.py

```diff
@@ -132,5 +249,11 @@ def serve(self):
         self.serve_event.wait()
         self.serve_event.clear()
 
+        while self.run:
+            sleep_interval = 5
+            self.log.debug('Sleeping for %d seconds', sleep_interval)
+            ret = self.event.wait(sleep_interval)
+            self.event.clear()
+
```

Comment:
i don't think we need this?

Reply:
it is copied from elsewhere. I've seen multiple ways of getthing the command to work. not sure which is the latest/ best
willl try without it